domain/domain.go

```diff
@@ -474,6 +496,7 @@ func NewDomain(store kv.Storage, ddlLease time.Duration, statsLease time.Duratio
 	// Only when the store is local that the lease value is 0.
 	// If the store is local, it doesn't need loadSchemaInLoop.
 	if ddlLease > 0 {
+		d.wg.Add(1)
```

Comment:
why need to wait for this rountine exit?

Reply:
Normally the structure stops, then the gorountine will exit.